docs/management.md

```diff
@@ -293,42 +299,63 @@ rabbitmqctl set_user_tags full_access administrator
 
 ## Authenticating with OAuth 2 {#oauth2-authentication}
 
-RabbitMQ can be configured to use [JWT-encoded OAuth 2.0 access tokens](https://github.com/rabbitmq/rabbitmq-auth-backend-oauth2) to authenticate client applications and management UI users. When doing so, the management UI does
-not automatically redirect users to authenticate
-against the OAuth 2 server, this must be configured separately. Currently,
-**Authorization code flow with PKCE** is tested with the following Authorization servers:
+RabbitMQ can be configured to use [JWT-encoded OAuth 2.0 access tokens](https://github.com/rabbitmq/rabbitmq-server/tree/main/deps/rabbitmq_auth_backend_oauth2) to authenticate client applications and management UI users. When doing so, the management UI does
+not automatically redirect users to authenticate against the OAuth 2 server, this must be configured separately. Currently, **Authorization code flow with PKCE** is tested with the following Authorization servers:
 
 * [UAA](https://github.com/cloudfoundry/uaa)
 * [Keycloak](https://www.keycloak.org/)
 * [Auth0](https://auth0.com/)
 * [Azure](https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/auth-oauth2)
 * [OAuth2 Proxy](https://oauth2-proxy.github.io/oauth2-proxy/)
+* [Okta](https://www.okta.com)
 
-**Important**: from the OAuth 2.0 point of view, the RabbitMQ Management UI is a **public app** which
-means it cannot securely store credentials such as the *client_secret*. This means that RabbitMQ does not need to present a client_secret when authenticating users.
 
-It is usually possible to configure the OAuth client as a **public app** with the authorization server that you are using.
-If target authorization server only supports a **confidential app** or it requires a *client_secret*,
-then a *client_secret* **must** be configured using the `oauth_client_secret` setting.
+### Basic configuration {#basic-configuration}
 
-To redirect users to the UAA server to authenticate, use the following configuration:
+Given the following configuration of the [OAuth 2.0 plugin](./oauth2):
+
+```ini
+auth_oauth2.resource_server_id = new_resource_server_id
+auth_oauth2.issuer = https://my-oauth2-provider.com/realm/rabbitmq
+```
+
+Apply the following configuration to enable OAuth 2.0 authentication:
 
```

Comment:
Verb usage is usually at the beginning of a sentence, I would change this to: "To enable OAuth 2.0 authentication, apply the following configuration":

Reply:
Updates for the "Verb usage" comment above are implemented, thanks @MarcialRosales 

What about the 2nd comment, please see screenshot below for reference, so the content is currently laid out like this: "To enable OAuth 2.0 authentication, apply the following configuration in the management UI assuming the following:" then there is 2 bullet points, a code block, and more bullet points. I presume the code block is the configuration they need to apply and then what is in the bullet points is what needs to be in place before applying the configuration so how about this instead: 

To enable OAuth 2.0 authentication, ensure the following configuration is complete: 

Include list of bullet here

Next, apply the following configuration

Include code block here